src/DataStream/Smoothing.h

```diff
@@ -11,7 +11,11 @@
 #ifndef CARTA_BACKEND__SMOOTHING_H_
 #define CARTA_BACKEND__SMOOTHING_H_
 
+#if _ARM_ // defined in cmake files
+#include "../third-party/sse2neon/sse2neon.h"
+#else
 #include <x86intrin.h>
+#endif // _ARM_
```

Comment:
our general approach with conditional `#include` entries is to move them below all other includes (for example, the boost FS stuff and the mach-o include